### ISSUE TITLE
failing test: test_generate_multi_value_hash()

### ISSUE THREAD

**Issue by trentmc** [oceanprotocol/ocean.py#273]
**Describe the bug**

Running
```console
pytest ocean_lib/web3_internal/test/test_utils.py::test_generate_multi_value_hash
```

Will cause
```console
AssertionError: assert '0x7ba270cc76...1b60662f60cea' == '0x6d59f15c58...00c0d736a9fec'
```

**To Reproduce**
1. Go through steps in developers.md
2. When running all tests with `pytest`, it fails on the above.

To reproduce faster, it's the test given above.

**Expected behavior**
Don't fail.

**Logs**

```console
(venv) trentmc@trentmcLnv:~/code/ocean.py$ pytest ocean_lib/web3_internal/test/test_utils.py
================================================================= test session starts ==================================================================
platform linux -- Python 3.8.8, pytest-6.2.3, py-1.10.0, pluggy-0.13.1
rootdir: /home/trentmc/code/ocean.py, configfile: pytest.ini
plugins: env-0.6.2
collected 2 items                                                                                                                                      

ocean_lib/web3_internal/test/test_utils.py F.                                                                                                    [100%]

======================================================================= FAILURES =======================================================================
____________________________________________________________ test_generate_multi_value_hash ____________________________________________________________

alice_address = '0xe2DD09d719Da89e5a3D0F2549c7E24566e947260'

    def test_generate_multi_value_hash(alice_address):
        with pytest.raises(AssertionError):
            generate_multi_value_hash(["more", "types", "than"], ["values"])
    
        expected = "0x6d59f15c5814d9fddd2e69d1f6f61edd0718e337c41ec74011900c0d736a9fec"
>       assert generate_multi_value_hash(["address"], [alice_address]).hex() == expected
E       AssertionError: assert '0x7ba270cc76...1b60662f60cea' == '0x6d59f15c58...00c0d736a9fec'
E         - 0x6d59f15c5814d9fddd2e69d1f6f61edd0718e337c41ec74011900c0d736a9fec
E         + 0x7ba270cc76c2dde25e744613ec459be48c8130f6f996b66f8df1b60662f60cea

ocean_lib/web3_internal/test/test_utils.py:17: AssertionError
---------------------------------------------------------------- Captured stdout setup -----------------------------------------------------------------
sender: 0xc594c6e5def4bab63ac29eed19a134c130388f74f019bc74b8f4389df2837a58, 0xe2DD09d719Da89e5a3D0F2549c7E24566e947260, None, address: 0xe2DD09d719Da89e5a3D0F2549c7E24566e947260
private key: 0xc594c6e5def4bab63ac29eed19a134c130388f74f019bc74b8f4389df2837a58
public key: 0x8b1b222564fb8302e9acfa4bcd60060de03ae479fcbed38fe21e45cd39ddb3fefb60ba1b4625e00d02223027c8f485f6d77693ab62ce06289b23942c6b319ccf

sender balance: 98.696703797
=============================================================== short test summary info ================================================================
FAILED ocean_lib/web3_internal/test/test_utils.py::test_generate_multi_value_hash - AssertionError: assert '0x7ba270cc76...1b60662f60cea' == '0x6d59f...
============================================================= 1 failed, 1 passed in 1.47s ==============================================================
```

**Comment by trentmc** [comment]
Note: 2476465b is the most commit on this test code

**Comment by trentmc** [comment]
Calina: it passes with tox and regular pytest, possibly there’s another exported value for test_private_key1

**Comment by trentmc** [comment]
(Trent): it doesn't pass with regular pytest for me

**Comment by DMats** [comment]
This is a duplicate of https://github.com/oceanprotocol/ocean.py/issues/256

**Comment by trentmc** [comment]
Closed, use #256
